docs/ref/models/database-functions.txt

```diff
@@ -262,7 +262,10 @@ Usage example::
     >>> authors = Author.objects.annotate(
     ...     smith_index=StrIndex('name', V('Smith'))
     ... ).order_by('smith_index')
-    >>> authors.first().smith_index
+    >>> first_author = authors.first()
```

Comment:
Perhaps this would be simpler:
```
>>> Author.objects.filter(name='Margaret Smith').annotate(
...     smith_index=StrIndex('name', V('Smith'))
... ).get().smith_index
```